cmd/ipfs-cluster-service/main.go

```diff
@@ -289,6 +299,25 @@ remove the %s file first and clean any Raft state.
 					cfgs.clusterCfg.Secret = userSecret
 				}
 
+				peersOpt := c.String("peers")
+				multiAddrs := []ma.Multiaddr{}
+				var peers []peer.ID
+				if peersOpt != "" {
+					addrs := strings.Split(peersOpt, ",")
+
+					if len(addrs) > 0 {
+						for _, addr := range addrs {
+							multiAddr, err := ma.NewMultiaddr(strings.TrimSpace(addr))
```

Comment:
I can do without trimming spaces, NewMultiaddr works well even if string contains spaces

Reply:
then do without

Reply:
my bad, I was wrong. 
tried like this `ipfs-cluster-service init --peers "   <multiaddress1>,   <multiaddress>"`, which failed because it doesn't start with `/`.